drivers/lora/Kconfig.rylrxxx

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2024 David Ullmann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config LORA_RYLRXXX
+	bool "Reyax LYLR driver"
+	default n
+	depends on DT_HAS_REYAX_RYLRXXX_ENABLED
+	select MODEM
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	select UART_USE_RUNTIME_CONFIGURE
+	help
+		Enable the Reyax LYLR driver
+
+config LORA_RYLRXX_RX_POOL_COUNT
```

Comment:
The options from here down should be guarded in a block like so:
```
if LORA_RYLRXXX

options go here...

endif # LORA_RYLRXXX
```
This prevents the options being configured when the modem isn't enabled